proposals/0029-cooperative-vector.md

```diff
@@ -142,7 +142,7 @@ declare <[NUMo] x [TYo] @dx.op.matvecmul.v[NUMo][TYo].v[NUMi][TYi](
     immarg i32,       ; matrix layout
     immarg i1,        ; matrix transpose
     i32,              ; matrix stride
-    immarg i1)        ; isResultSigned        <<< See #399
+    immarg i8)        ; flags
```

Comment:
I think i32 for a flags field would be more typical/appropriate, even if we don't imagine burning through those bits.  I just don't think there's any real advantage to using i8 for flags instead.

Additionally, it would help to identify the flag set in the comment here, for instance by the enum name or prefix in DxilConstants.h (`kMatVecMulFlag_*` at the moment), unless we have a better idea.